storage/bucket.go

```diff
@@ -225,6 +225,9 @@ func (b *BucketHandle) SignedURL(object string, opts *SignedURLOptions) (string,
 // to be non-nil. You may need to set the GoogleAccessID and PrivateKey fields
 // in some cases. Read more on the [automatic detection of credentials] for this method.
 //
+// To support POSTs of dynamic objects, pass an empty string to `object` and provide a condition for `$key`.
+// For example: `storage.ConditionStartsWith("$key", "prefix")`
```

Comment:
I am not sure if "dynamic objects" is clear phrasing. Maybe: `To allow the unauthenticated client to upload to any object name in the bucket with a given prefix rather than a specific object name, you can pass an empty string for object and set [PostPolicyV4Options].Conditions to include [ConditionStartsWith]("$key", "prefix").` Does that sound right?

Also, the backticks are not supported in godoc formatting; you can remove them.

Reply:
Thanks, that sounds perfect.